docs/_user_guide/storage.md

```diff
@@ -1,276 +1,180 @@
+ retired the Box service and has taken steps to [migrate Box content](https://uit.stanford.edu/project/box-migration){:target="_blank"} to a new folder on Google Drive or Microsoft OneDrive.
 
-Storing data in the cloud is an effective way to inexpensively archive data, serve files publicly, and integrate with cloud-native query and compute tools. With the growing number of cloud storage options and security risks, we advise caution when choosing to store your data on any cloud platform. If you are considering cloud solutions for storage, please contact [DARC](http://darcrequest.stanford.edu) to discuss your needs.
+## Cloud Platforms
 
+Storing data in the cloud is an effective way to inexpensively archive data, serve files publicly, and integrate with cloud-native query and compute tools. With the growing number of cloud storage options and security risks, we advise caution when choosing to store your data on any cloud platform. If you are considering cloud solutions for storage, please contact [DARC](mailto:gsb_darcresearch@stanford.edu) to discuss your needs.
 
-## File Storage
+## Yen File System
 
-You have several options for where to store your research files (data sets, programs, output files, and so forth). This guide will help you decide which storage location is best for your situation.
+The GSB offers nearly 1 PB of high-performance storage via the Yen servers, accessible at the ```/zfs``` path. Currently, all Yen user directories (``` ~/```) reside on ZFS. In addition to home directories, Stanford GSB faculty—and in some cases, students (see below)—can request additional project space on ZFS.
 
-## Yen File System 
+!!! Warning
+    The Yen servers are **not** approved for high risk data. ZFS is mounted **only** from the Yen servers. You cannot access it from Sherlock, FarmShare or any other system.
 
-If you are new to using the Unix shell, please go over the <a href="/gettingStarted/shell_novice.html" target="_blank">Shell Introduction</a>
-first.
- 
-If you’re already comfortable manipulating files and directories (using `ls`, `pwd`, `cd`, `mv`, `rm` commands), 
-you probably want to explore the next lesson: <a href="/gettingStarted/shell_extras.html" target="_blank">Shell Extras</a>, 
-to learn about searching for files with `grep` and `find`, and writing simple shell loops and scripts.
- 
 ### Home Directory
-Every user on the Yens has a home directory. This is where you are when you login to the system.
+
+Every user on the Yens has a home directory. Home directories on the Yen servers are also stored on our storage system. The home directory has a quota of 50 GB.
+
+When you log into the Yen servers, you will automatically land in your home directory, which is located at ```/home/users/<SUNet ID>``` or with the shortcut `~`. You are able to access your former AFS home directory by following the ```afs-home``` symlink inside your home directory.
+
 Check the absolute path with:
 
-```bash
-$ pwd
+```title="Terminal Command"
```

Comment:
Please disable the copy button by doing. Please repeat this for all other commands. {.yaml .no-copy title="Terminal Command"} with ``` in front